test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportDataValidationTests.cs

```diff
@@ -90,6 +90,58 @@ public async Task GivenFhirServer_WhenPatientDataIsExported_ThenExportedDataIsSa
             Assert.True(ValidateDataFromBothSources(dataFromFhirServer, dataFromExport));
         }
 
+        [Fact]
+        public async Task GiveFhirServer_WhenAllObservationDataIsExported_ThenExportedDataIsSameAsDataInFhirServer()
```

Comment:
Test name says `AllObservation` whereas `_type` param includes Observation and Patient

Reply:
Woops. I wanted to have the test show that sending multiple types worked, but forgot to update the test name.